release_notes.md

```diff
@@ -1,5 +1,7 @@
 ### Version 0.0.19 ###
 
+* Added Adam optimizer
+* GradientOptimizer and NonGradientOptimizer now takes an algorithm argument that by default will use Conjugate Gradient and BOBYQA respectively.
```

Comment:
can you call out the breaking changes more explicitly, e.g. the `GradientOptimizer` returns a new class of object `OptimizedResult`

Reply:
👍